package.json

```diff
@@ -50,12 +50,12 @@
     "underscore": "~1.8.3"
   },
   "devDependencies": {
-    "babel-core": "6.17.0",
+    "babel-core": "^6.24.0",
```

Comment:
I think this is fine, though now that we're using yarn the `^` just indicates how `yarn upgrade` works. The lock file should keep it from changing versions.  As long as everyone uses yarn that is.